webui/react/src/pages/Cluster/ClusterOverallStats.tsx

```diff
@@ -7,8 +7,6 @@ import { useStore } from 'contexts/Store';
 import { ShirtSize } from 'themes';
 import { ResourceType } from 'types';
 
-import { maxPoolSlotCapacity } from '../Clusters/ClustersOverview';
-
```

Comment:
**question:** I noticed that `maxPoolSlotCapacity` is no longer used in the `ClusterOverallStats` and `ClusterOverallBar` display. There's two more places that reference that function.
1. `ClusterOverview` - we talked about a dedicated allocation endpoint in the future, but for now will the function `clusterStatusText` be incorrect and also double count agents/slots since it is still relying on `maxPoolSlotCapacity`?
2. `ResourcePoolCardLight` - this one might be ok since it's showing stats on a per pool basis, but should it use the new overview agents data instead also?

Reply:
1. yeah.. I think it would be incorrect. good catch. let me give this some thought and see what I come up with.
2. yeah, I intentionally didn't update it, but after more thought I agree, it should. will update.

Reply:
I guess the change I made has issues when using dynamic agents. this may get a bit more tricky than I first thought.

Reply:
ok, so.
1. I corrected the calculation for `maxClusterSlotCapacity` for all cases and pulled in into a function and reused it for `clusterStatusText`
2. I misread this question - since it is per pool I think it is fine still.